test/e2e/scripts/cleanup.ts

```diff
@@ -38,7 +59,8 @@ async function run() {
     if (
       new Date().getTime() - createdAt.getTime() > DATASET_MAX_AGE &&
       dataset.name.startsWith('pr-') &&
-      !dataset.name.endsWith('-comments')
+      !dataset.name.endsWith('-comments') &&
+      dataset.name !== SANITY_E2E_DATASET_STAGING
```

Comment:
don't think this can ever happen unless `SANITY_E2E_DATASET_STAGING` starts with `pr-`?